modules/ROOT/pages/creating-and-managing-a-cluster-manually.adoc

```diff
@@ -124,19 +124,37 @@ Two tables are created per object store:
 
 ==== Recommendations for the Object Store Database
 
-. It is recommended that you create a dedicated database/schema that will only be used for JDBC store.
-. The database username needs to have permission to:
-.. Create objects in the database, which means DDL CREATE, DROP for tables.
-.. DML permissions on the objects it creates (INSERT, UPDATE, DELETE, SELECT)
-. Always keep in mind that the data storage needs to be hosted in a centralized DB reachable from all nodes. Don't use more than one database per cluster. +
+* It is recommended that you create a dedicated database/schema that will only be used for JDBC store.
+* The database username needs to have permission to:
+** Create objects in the database, which means DDL CREATE, DROP for tables.
+** DML permissions on the objects it creates (INSERT, UPDATE, DELETE, SELECT)
+* Always keep in mind that the data storage needs to be hosted in a centralized DB reachable from all nodes. Don't use more than one database per cluster. +
 Check the <<persistency-config,cluster configuration reference for persistency>> for details on how to configure these values.
+* Some relational databases have certain constraints regarding the name length of tables. To leap over this, use the *mule.cluster.jdbcstoretableNametransformerstrategy* property. +
+Check the <<table-name-transformers, Table Name Transformers>> section for more details on how to configure this property.
 
+[[table-name-transformers]]
+===== Table Name Transformers
+
+The *mule.cluster.jdbcstoretableNametransformerstrategy* property allows you to define a custom transformer to modify table names.
+
+For example, if you set the following property in `mule-cluster.properties`:
+[source]
+----
+...
+mule.cluster.jdbcstoretableNametransformerstrategy=com.mulesoft.mule.cluster.hazelcast.persistence.query.MD5TableNameTransformerStrategy
+...
+----
+The table names will be hashed using MD5 and a prefix to identify that these are Mule tables. Hashing table names guarantee that the length constraint is honored.
+
+Optionally, you can create a custom transformer strategy by implementing the interface
+`com.mulesoft.mule.cluster.hazelcast.persistence.query.TableNameTransformerStrategy`
 
 === Monitoring
 
 You can monitor the events thrown by the cluster members through the http://www.oracle.com/technetwork/java/javase/tech/javamanagement-140525.html[JMX technology].
 
-The JMX monitoring option is disabled by default. In order to enable it, you need to add the +mule.cluster.jmxenabled+ property to the +{MULE_HOME}/.mule/mule-cluster.properties+ configuration file.
+The JMX monitoring option is disabled by default. In order to enable it, you need to add the `mule.cluster.jmxenabled` property to the `{MULE_HOME}/.mule/mule-cluster.properties` configuration file.
```

Comment:
recommendation to eliminate unnecessary words: 
"In order to enable" -> "To enable"